### PR TITLE
Suisin/chore: add Sinhalese language in SmartCharts

### PR DESCRIPTION
**Pull request by suisin-deriv**
# Description
Add Sinhalese language into SmartCharts

# Cards
* https://app.clickup.com/t/20696747/UPM-902)

### CODE REVIEW

**Review by review-deriv — COMMENTED** [review]
PR description is too short. You can use this format:

```
# Description
This is a summary of the changes. This should describe why do we need this change.
Whys and Hows come here.

# Cards
* https://clickup.com/12345
* https://clickup.com/12346

```

**Review by likhith-deriv — APPROVED** [review]

**Review by yauheni-deriv — APPROVED** [review]

**Review by amina-deriv — COMMENTED** [review]
Flag for arabic seems different from app.deriv.com  Is this expected?
Rest is LGTM

File: src/Constant.tsx [hunk 1]
```diff
@@ -1031,8 +1031,8 @@ const getSMIIndicatorConfig: TDefaultIndicatorConfigFn = () => ({
         pipSize: 2,
         showLastIndicator: true,
         smiOscillatorLimits: {
-            overboughtStyle: { ...clone(lineStyle), color: '#626b73'},
-            oversoldStyle: { ...clone(lineStyle), color: '#626b73'},
+            overboughtStyle: { ...clone(lineStyle), color: '#626b73' },
+            oversoldStyle: { ...clone(lineStyle), color: '#626b73' },
```
Comment (henry-deriv) [inline src/Constant.tsx:1035]: this is not relevant

**Review by henry-deriv — APPROVED** [review]

**Review by amina-deriv — APPROVED** [review]

**Review by fasihali-deriv — APPROVED** [review]

**Review by ahmadtaimoor-deriv — APPROVED** [review]

**Review by maryia-deriv — APPROVED** [review]

**Pull request merged**